src/studio/src/designer/frontend/ux-editor/features/appData/appDataReducers.ts

```diff
@@ -0,0 +1,21 @@
+import { combineReducers, Reducer } from 'redux';
+import dataModelReducer, { IDataModelState } from './dataModel/dataModelSlice';
+import languageReducer, { ILanguageState } from './language/languageSlice';
+import ruleModelReducer, { IRuleModelState } from './ruleModel/ruleModelSlice';
+import textResourceReducer, { ITextResourcesState } from './textResources/textResourcesSlice';
+
+export interface IAppDataState {
+  dataModel: IDataModelState;
+  textResources: ITextResourcesState;
+  ruleModel: IRuleModelState;
+  language: ILanguageState;
```

Comment:
Suggest finding a way to avoid this `language.language`. Perhaps `languageState.language`

Reply:
fixed.